filebeat/module/haproxy/log/test/tcplog.log-expected.json

```diff
@@ -0,0 +1,30 @@
+[
+    {
+        "@timestamp": "2018-09-20T15:44:23.285Z", 
+        "fileset.module": "haproxy", 
+        "fileset.name": "log", 
+        "haproxy.backend_name": "app", 
+        "haproxy.backend_queue": 0, 
+        "haproxy.bytes_read": 212, 
+        "haproxy.client.ip": "127.0.0.1", 
+        "haproxy.client.port": 40962, 
+        "haproxy.connection_wait_time_ms": -1, 
+        "haproxy.connections.active": 1, 
+        "haproxy.connections.backend": 0, 
+        "haproxy.connections.frontend": 1, 
+        "haproxy.connections.retries": 0, 
+        "haproxy.connections.server": 0, 
+        "haproxy.frontend_name": "main", 
+        "haproxy.pid": 25457, 
+        "haproxy.process_name": "haproxy", 
+        "haproxy.server_name": "<NOSRV>", 
+        "haproxy.server_queue": 0, 
+        "haproxy.source": "127.0.0.1", 
+        "haproxy.tcp.processing_time_ms": 0, 
+        "haproxy.termination_state": "SC", 
+        "haproxy.total_waiting_time_ms": -1, 
```

Comment:
Just curiosity, do you know what these `-1`s mean?

Reply:
_the total time in milliseconds spent waiting for the connection to establish to the final server... It can be "-1" if the connection was aborted before a connection could be established..._ In the example log I'm using it's actually a -1, which is a bit weird because it was taken with my own HAProxy setup

Reply:
Oh, thanks